app/components/Showcases/Barter.jsx

```diff
@@ -0,0 +1,724 @@
+                        to_account,
+                        to_balanceError,
+                        to_asset_types
+                    )}
+                    allowNaN={true}
+                />
+                <div style={{paddingTop: "10px", paddingBottom: "10px"}}>
+                    <Button onClick={this.addToAmount.bind(this)}>
+                        + Add asset
+                    </Button>
+                </div>
+                {toAmountSelector}
+            </Card>
+        );
+
+        let offers = (
+            <Card style={{borderRadius: "10px"}}>
+                {from_name} offers C X to {to_name} and receives D Y in return,
+                implict price is:
+                <Input addonAfter="X/Y" />
+            </Card>
+        );
+
+        return (
+            <div
+                className="container no-overflow wrap shrink"
+                style={{padding: "10px"}}
+            >
+                <Card>
+                    {smallScreen ? (
+                        <div>
+                            <Row>
+                                <Col style={{padding: "10px"}}>
+                                    {account_from}
+                                </Col>
+                            </Row>
+                            <Row>
+                                <Col style={{padding: "10px"}}>
+                                    {account_to}
+                                </Col>
+                            </Row>
+                            <Row>
+                                <Col style={{padding: "10px"}}>{offers}</Col>
+                            </Row>
+                        </div>
+                    ) : (
+                        <div>
+                            <Row>
+                                <Col span={12} style={{padding: "10px"}}>
+                                    {account_from}
+                                </Col>
+                                <Col span={12} style={{padding: "10px"}}>
+                                    {account_to}
+                                </Col>
+                            </Row>
+                            <Row>
+                                <Col style={{padding: "10px"}}>{offers}</Col>
+                            </Row>
+                        </div>
+                    )}
+                    <Button
+                        key={"send"}
+                        disabled={isSubmitNotValid}
+                        onClick={
+                            !isSubmitNotValid ? this.onSubmit.bind(this) : null
+                        }
+                    >
+                        {counterpart.translate("propose")}
+                    </Button>
+                </Card>
+            </div>
+        );
+    }
+}
+
+Barter = debounceRender(Barter, 50, {leading: false});
+
+class BarterWrapper extends React.Component {
+    render() {
+        return <Barter {...this.props} />;
+    }
+}
+
+export default connect(
```

Comment:
Please remove unused Stores and imports

Reply:
I deleted unused Stores(SettingsStore, MarketsStore, AssetStore) and imports(SettingsStore, AssetStore, debounceRender, connect, MarketsStore)